Pipfile

```diff
@@ -17,6 +17,7 @@ sklearn = "*"
 sphinx = "*"
 sphinx-rtd-theme = "*"
 toml = "*"
+nbsphinx = "*"
```

Comment:
I see we have many packages that maybe should be in the `dev-packages` section? I am not sure though. What do you think? I am not 100% sure if these documentation packages should be dev-dependencies or not.

Reply:
For now I think it should be ok to move them to `dev-packages`.
At some point, we might be able to generate `dev` and `prod` documentation, so we would need both packages there.

How to proceed with this?

Reply:
@felipehuerta17 @leandrolanzieri any opinions on this?

Reply:
I am not familiar what's good practice in this domain. For the moment, I don't have a problem on having a long list, as the user can still do `git clone, cd open-sir and pip install.`. And I think that completeness and accuracy is priority over conciseness, at this stage.

Reply:
ok, I propose this: let's proceed with this one and I open an issue to decide the boundaries between pipenv (dev, prod) and setuptools (pip install opensir).

Does this sound reasonable?